tests/apps/test_similarity.py

```diff
@@ -222,217 +223,474 @@ def test_event_cardinality(photons, max_count, modes, expected):
+        assert similarity.prob_orbit_mc(graph, [], 0) == 1.0
 
-    def test_all_loss(self, monkeypatch):
+    def test_max_loss(self):
         """Test if function samples from the vacuum when maximum loss is applied."""
-        dim = 5
-        graph = nx.complete_graph(dim)
-        mock_eng_run = mock.MagicMock()
-
-        with monkeypatch.context() as m:
-            m.setattr(sf.LocalEngine, "run", mock_eng_run)
-            similarity.prob_orbit_mc(graph, [1, 1, 1, 1], samples=1, loss=1)
-            p_func = mock_eng_run.call_args[0][0]
-
-        eng = sf.LocalEngine(backend="gaussian")
-
-        state = eng.run(p_func).state
-        cov = state.cov()
-        disp = state.displacement()
+        graph = nx.complete_graph(5)
+        assert similarity.prob_orbit_mc(graph, [1, 1, 1, 1], samples=1, loss=1) == 0.0
 
-        assert np.allclose(cov, 0.5 * state.hbar * np.eye(2 * dim))
-        assert np.allclose(disp, np.zeros(dim))
+    def test_known_result(self):
+        """Tests if the probability of detecting photons in nearly all modes of a
+        graph with a small mean photon number is correctly reproduced as
+        almost zero."""
+        graph = nx.complete_graph(4)
+        assert np.allclose(similarity.prob_orbit_mc(graph, [4], 1), 0)
 
 
 class TestProbEventMC:
-    """Tests for the function ``strawberryfields.apps.similarity.prob_event_mc.``"""
+    """Tests for the function ``strawberryfields.apps.similarity.prob_event_mc``"""
 
     def test_invalid_samples(self):
         """Test if function raises a ``ValueError`` when a number of samples less than one is
         requested."""
-        g = nx.complete_graph(10)
+        g = nx.complete_graph(5)
         with pytest.raises(ValueError, match="Number of samples must be at least one"):
             similarity.prob_event_mc(g, 2, 2, samples=0)
 
     def test_invalid_n_mean(self):
         """Test if function raises a ``ValueError`` when the mean photon number is specified to
         be negative."""
-        g = nx.complete_graph(10)
+        g = nx.complete_graph(5)
         with pytest.raises(ValueError, match="Mean photon number must be non-negative"):
             similarity.prob_event_mc(g, 2, 2, n_mean=-1)
 
     def test_invalid_loss(self):
         """Test if function raises a ``ValueError`` when the loss parameter is specified outside
         of range."""
-        g = nx.complete_graph(10)
-        with pytest.raises(ValueError, match="Loss parameter must take a value between zero and"):
+        g = nx.complete_graph(5)
+        with pytest.raises(ValueError, match="Loss parameter must take a value between zero and one"):
             similarity.prob_event_mc(g, 2, 2, loss=2)
 
     def test_invalid_photon_number(self):
         """Test if function raises a ``ValueError`` when a photon number below zero is specified"""
-        g = nx.complete_graph(10)
+        g = nx.complete_graph(5)
```

Comment:
It's not a big problem, but it would be good to minimize doing changes like these to reduce the overhead of code review. Unless of course the changes are needed/improve something!